org.adempiere.base/src/org/compiere/model/MTable.java

```diff
@@ -601,6 +601,21 @@ public PO getPO (ResultSet rs, String trxName)
 		return po;
 	}	//	getPO
 
+	/**
+	 * Get PO Class Instance
+	 * 
+	 * @param  uuID    UUID
+	 * @param  trxName transaction
+	 * @return         PO for Record
+	 */
+	public PO getPOByUU (String uuID, String trxName)
+	{
+		PO po = getPO(0, trxName);
+		po.loadByUU(uuID, trxName);
+
+		return po;
+	} // getPOByUU
+

```

Comment:
Since we added new method to MTable, we need to update the serialVersionUID of MTable.

Reply:
Done